source/uwp/Renderer/dll/CppWinRTIncludes.h

```diff
@@ -34,18 +43,21 @@ namespace winrt
     using namespace ::winrt::Windows::Foundation::Numerics;
     using namespace ::winrt::Windows::Storage::Streams;
     using namespace ::winrt::Windows::System;
-    using namespace ::winrt::Windows::UI;
-	using namespace ::winrt::Windows::UI::Xaml;
-    using namespace ::winrt::Windows::UI::Xaml::Controls;
-    using namespace ::winrt::Windows::UI::Xaml::Controls::Primitives;
-    using namespace ::winrt::Windows::UI::Xaml::Automation;
-	using namespace ::winrt::Windows::UI::Xaml::Automation::Peers;
+    using namespace ::winrt::AdaptiveCards::ObjectModel::Uwp;
+
+    // WinUI 2
+    using namespace ::winrt::Microsoft::UI;
+	using namespace ::winrt::Microsoft::UI::Xaml;
```

Comment:
it would be good to standardize this straddling of system xaml and winui3 across our UI frameworks as I expect others need this too. I've done this already here: https://github.com/asklar/xamlstraddle